packages/cli/src/commands/build.js

```diff
@@ -93,24 +80,73 @@ export const handler = async ({
     )
   }
 
-  const tasks = new Listr(
-    side.map((sideName) => {
-      const { cwd, cmd } = execCommandsForSides[sideName]
-      return {
-        title: `Building "${sideName}"${(stats && ' for stats') || ''}...`,
+  const listrTasks = side.map((sideName) => {
+    const { cwd, cmd } = execCommandsForSides[sideName]
+    return {
+      title: `Building "${sideName}"${(stats && ' for stats') || ''}...`,
+      task: () => {
+        return execa(cmd, undefined, {
+          stdio: verbose ? 'inherit' : 'pipe',
+          shell: true,
+          cwd,
+        })
+      },
+    }
+  })
+
+  // Additional tasks, apart from build
+  if (side.includes('api')) {
+    try {
+      await generatePrismaClient({
+        verbose,
+        force: true,
+        schema: getPaths().api.dbSchema,
+      })
+    } catch (e) {
+      console.log(c.error(e.message))
+      process.exit(1)
+    }
+
+    listrTasks.unshift({
+      title: 'Cleaning api/dist',
```

Comment:
We actually pass `--delete-dir-on-start` when building the api side, is this required?

Reply:
Hmmm good point, not sure it used to work for me though, haven't checked it recently.

Reply:
Veritifed. Removing this bit now